src/index.html

```diff
@@ -11,7 +11,155 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset class="field">
+        <legend>Personal information</legend>
+        <label for="surname" class="field-forms">
+          Surname:
+          <input
+          autocomplete="off"
+          type="text"
+          id="surname"
+          minlength="5"
+          maxlength="16"
+          required
+          >
```

Comment:
let's fix attributes formatting everywhere, [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) 3rd item
```suggestion
          <input
            autocomplete="off"
            type="text"
            id="surname"
            minlength="5"
            maxlength="16"
            required
          >
```